basic-application/basic-application-core/src/main/java/fr/openwide/core/basicapp/core/business/user/service/UserServiceImpl.java

```diff
@@ -54,17 +45,32 @@ private void audit(User subject, User object, AuditAction action, String methodN
 
 	@Override
 	public void onSignIn(User user) throws ServiceException, SecurityServiceException {
-		audit(user, AuditAction.SIGN_IN, AUDIT_SIGN_IN_METHOD_NAME);
+		historyLogService.log(HistoryEventType.SIGN_IN, HistoryLogObjectsBean.of(user));
 	}
 
 	@Override
 	public void onSignInFail(User user) throws ServiceException, SecurityServiceException {
-		audit(user, user, AuditAction.SIGN_IN_FAIL, AUDIT_SIGN_IN_FAIL_METHOD_NAME);
+		historyLogService.log(HistoryEventType.SIGN_IN_FAIL, HistoryLogObjectsBean.of(user));
 	}
 
 	@Override
 	public void onCreate(User user, User author) throws ServiceException, SecurityServiceException {
-		audit(author, user, AuditAction.CREATE_USER, AUDIT_CREATE_METHOD_NAME);
+		historyLogService.log(HistoryEventType.CREATE, HistoryLogObjectsBean.of(user));
+	}
+	
+	@Override
+	public void setActive(User person, boolean active) throws ServiceException, SecurityServiceException {
+		super.setActive(person, active);
+		historyLogService.log(active ? HistoryEventType.ENABLE : HistoryEventType.DISABLE, HistoryLogObjectsBean.of(person));
+	}
+	
+	@Override
+	protected void updateEntity(User person) throws ServiceException, SecurityServiceException {
+		super.updateEntity(person);
+		
+//		historyLogService.logWithDifferences(HistoryEventType.UPDATE, HistoryLogObjectsBean.of(person),
```

Comment:
I'm okay with commenting this line so as not to include unnecessary features in new applications, but maybe we could just add a call to `.log(HistoryEventType.UPDATE, HistoryLogObjectsBean.of(person))` (without differences)?